client/src/pages/Authentication/CustomerRegister.js

```diff
@@ -24,94 +41,153 @@ const Register = () => {
-										
-										value={name}
-										onChange={(e) => setName(e.target.value)}
-										type="text"
-										placeholder="Name"
-									/>
-								</div>
-								<div className="mb-6">
-									<input
-
-										className="form-control block w-full px-4 py-2 text-xl font-normal text-gray-700 bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
-										
-										placeholder="Email address"
-										value={email}
-										onChange={(e) => setEmail(e.target.value)}
-										type="email"
-
-									/>
-								</div>
-
-								<div className="mb-6">
-									<input
-										className="form-control block w-full px-4 py-2 text-xl font-normal text-gray-700 bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
-										
-										placeholder="Password"
-										value={password}
-										onChange={(e) => setPassword(e.target.value)}
-										type="password" />
-								</div>
-
-							
-
-								<div className="text-center lg:text-left">
-									<button
-										type="submit"
-										className="inline-block px-7 py-3 bg-blue-600 text-white font-medium text-sm leading-snug uppercase rounded shadow-md hover:bg-blue-700 hover:shadow-lg focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out"
-									>
-										Register
-									</button>
-									<p className="text-md font-semibold mt-2 pt-1 mb-0">
-										Have an account?
-										<a
-											href="/login"
-											className="text-red-600 hover:text-red-700 focus:text-red-700 transition duration-200 ease-in-out"
-										> Login</a
+							<h1 className='text-center font-light mt-2 text-3xl  border-red-400 '>Signup</h1>
+
+							<div className="mb-0 md:mb-0  border-white mx-auto p-8 w-full">
+
+
+
+								<form onSubmit={registerUser} className=' border-white'>
+
+									<div className="mb-[0.75rem]  border-white">
+										<div className="text-white text-left ml-4 text-xl mb-[0.8rem]">
+											Name
+										</div>
+										<input
+
+											className={`form-control block xl:w-96 px-4 py-2 text-xl font-normal text-white bg-AuthInput bg-clip-padding rounded-full transition ease-in-out m-0 focus:text-gray-700 focus:bg-gray-800 focus:outline-none border-none`}
+
+											placeholder="Username"
+											value={name}
+											onChange={(e) => OnnameChange(e)}
+											type="text"
+										/>
+										<div className={`text-red-400 text-left mt-1 d-${namevalidator}`}>
+											* username must have for than 10 chararcter
```

Comment:
Your username validator is not working correctly. The warning is removed after entering 12 characters.